reporters/kamon-datadog/src/main/scala/kamon/datadog/DatadogSpanReporter.scala

```diff
@@ -52,7 +53,14 @@ object KamonDataDogTranslatorDefault extends KamonDataDogTranslator {
     val start = from.getEpochNano
     val duration = Duration.between(from, span.to)
     val marks = span.marks.map { m => m.key -> m.instant.getEpochNano.toString }.toMap
-    val tags = (span.tags.all() ++ span.metricTags.all() ++ additionalTags.all()).map { t =>
+    val errorTags = if (span.hasError) {
+      val builder = TagSet.builder()
+      span.tags.get(Lookups.option(TagKeys.ErrorMessage)).foreach(msg => builder.add("error.msg", msg))
```

Comment:
DataDog seem to use `error.message` too, so think this is redundant (cf [the docs](https://docs.datadoghq.com/tracing/error_tracking/#use-span-tags-to-track-error-spans))